package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@jothepro/doxygen-awesome-css",
+  "version": "2.2.1-pre",
+  "description": "Custom CSS theme for doxygen html-documentation with lots of customization parameters.",
+  "main": "",
+  "scripts": {
+    "npm-pack": "npm pack",
+    "postversion": "git push origin --all && git push origin --tags",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jothepro/doxygen-awesome-css.git"
+  },
+  "homepage": "https://jothepro.github.io/doxygen-awesome-css/",
+  "bugs": {
+    "url": "https://github.com/jothepro/doxygen-awesome-css/issues"
+  },
+  "keywords": [
+    "doxygen",
+    "css",
+    "theme",
+    "awesome"
+  ],
+  "author": {
+    "name": "jothepro",
+    "url": "https://github.com/jothepro",
+    "git": "https://github.com/jothepro/doxygen-awesome-css"
+  },
+  "license": "MIT",
+  "config": {},
+  "dependencies": {},
+  "devDependencies": {},
+  "xpack": {}
```

Comment:
Are these empty keys required to be present?

Reply:
The `xpack` object is necessary to make the package also an xPack, i.e. installable with `xpm`.

The other are not mandatory, but generally seeing them empty makes obvious for everybody that the package has no other dependencies.

If needed, they can be removed.

Reply:
I have no opinion on that. It's fine for me to have the empty objects if you consider this good practice. :+1: